rs/nns/governance/src/governance.rs

```diff
@@ -94,7 +94,11 @@ use ic_nns_constants::{
     SUBNET_RENTAL_CANISTER_ID,
 };
 use ic_nns_governance_api::{
-    pb::v1::CreateServiceNervousSystem as ApiCreateServiceNervousSystem, proposal_validation,
+    pb::v1::{
+        self as api, CreateServiceNervousSystem as ApiCreateServiceNervousSystem, ListNeurons,
+        ListNeuronsResponse,
```

Comment:
In addition to bringing in the analogous API types, this also does

```
self as api
```

I recommend that we do this to reduce confusion in a world where we have many times with the exact same name.